setup.py

```diff
@@ -26,7 +26,7 @@
     'passlib',
     'pycryptodome>=3.6.6',  # security update
     'PyGithub>=1.43.3',
-    'pyOpenSSL~=23.0',
+    'pyOpenSSL~=23.2',
```

Comment:
I see no references to `OpenSSL` in our codebase. Is this a necessary dependency anymore?

Reply:
Great question. Let's try removing it and see what it breaks.

Reply:
Update: It looks like Marco added this in d1810fbbbb85ab4bddc744de2e4ef27caf6d4fdc because we had previously been relying on a system package that gets installed indirectly. I think we need to keep it.